packages/retail-ui-extensions/src/extension-api/toast-api/toast-api.ts

```diff
@@ -0,0 +1,15 @@
+export interface ToastApiContent {
```

Comment:
I see this API has a different shape than the one used in [admin-ui-extensions](https://shopify.dev/docs/api/admin-extensions/api/toastapi) and other surfaces.

To reduce fragmentation, could we use something closer to Admin?

```ts
export interface ShowToastOptions {
  duration?: number;
}

export interface ToastApiContent {
  show: (content: string, options?: ShowToastOptions) => void;
}
```